comet-java-client/src/main/java/ml/comet/experiment/impl/asset/Asset.java

```diff
@@ -0,0 +1,42 @@
+package ml.comet.experiment.impl.asset;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ml.comet.experiment.context.ExperimentContext;
+
+import java.io.File;
+import java.util.Map;
+
+/**
+ * Describes asset data.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Asset {
```

Comment:
for non mandatory fields (step, epoch) I think its cleaner to use null as default (and use non primitives)